### ISSUE TITLE
install_native.sh doesn't work on macos

### ISSUE THREAD

**Issue by raff** [YijinLiu/headless-chromium#1]
It assumes options (readlink -f) and tools (apt) that are only available on certain versions of Linux.

I'll submit a pull request, at least for macos, once I have something working.

**Comment by lox** [comment]
Where did you get up to @raff? I'd be happy to help.

**Comment by raff** [comment]
Well, "gclient sync"

**Comment by raff** [comment]
Ok, let me try again...

I made a couple of changes to the scripts and now "gclient sync" starts but
never finishes (well, after a very long time it ends with an error)

I am trying again now...

-- Raffaele

**Comment by YijinLiu** [comment]
Sorry for slow response. Please note that this project doesn't work yet. Chromium needs to start a zygote process to render HTML pages, which requires the process don't start any thread. Golang by nature is multiple threads. Chrome team have fixed that by https://codereview.chromium.org/2384163002
I haven't tried that yet.

**Comment by lox** [comment]
No problems @YijinLiu, presumably it will eventually work!
